### DISCUSSION TITLE
Why does the `flag` module not include `-v` for showing the version (`--version`)

### DISCUSSION THREAD

**Discussion by yochem** [vlang/v#16737] [Questions and Answers] [answered]
As the title of the question, Why does the `flag` module not include `-v` for showing the version (`--version`). It does so with the standard help option: `-h` and `--help`. Since `-v` is used quite often for showing the version of a program, I would expect it to also be included in the default flags. If there is interest in this, I can make a PR implementing this.

**Comment by JalonSolov** [comment, marked as answer]
Because `-v` is often used to mean `--verbose`.  Since it's not 100% standard, it's up to you to add the flag yourself, for whichever thing you want it to mean.

**Reply by yochem** [reply]
Ah right, did not think of --verbose. Thanks for the explanation.
